contrib/scripts/bootc-destructive-cleanup

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -xeuo pipefail
+
+mount -o remount,rw /sysroot
+rpm -qa --root=/sysroot --dbpath=/usr/lib/sysimage/rpm | xargs rpm -e --root=/sysroot --dbpath=/usr/lib/sysimage/rpm
+
+mount --bind -o rw /sysroot/var/lib/containers /var/lib/containers
```

Comment:
Also I guess conceptually since podman is OS independent in theory this could be a shared fragment, but let's not overengineer now, just noting.